internal/ingress/controller/template/template.go

```diff
@@ -588,6 +595,33 @@ func buildAuthResponseHeaders(proxySetHeader string, headers []string) []string
 	return res
 }
 
+func buildAuthUpstreamHeaders(proxySetHeader string, headers []string) []string {
+	res := []string{}
+
+	if len(headers) == 0 {
+		return res
+	}
+
+	for i, h := range headers {
+		res = append(res, fmt.Sprintf("set $authHeader%v '';", i))
```

Comment:
i is int, h is string, so let's stick with %d here (for the int) and %s below as string, and avoid some "weird leak" in a future